### ISSUE TITLE
NullPointException if known_host is missing

### ISSUE THREAD

**Issue by arichiardi** [flosell/lambdacd-git#21]
Hello folks!
I am trying this package for cloning my repo and I have run into a NPE:

```
NullPointerException 
java.io.SequenceInputStream.nextStream (SequenceInputStream.java:112)
java.io.SequenceInputStream.<init> (SequenceInputStream.java:69)
lambdacd-git.core/set-known-hosts/fn--13639 (core.clj:216)
lambdacd-git.ssh-agent-support/session-factory/fn--13541/fn--13544 (ssh_agent_support.clj:33)
clojure.core/map/fn--4785 (core.clj:2644)
clojure.lang.LazySeq.sval (LazySeq.java:40)
clojure.lang.LazySeq.seq (LazySeq.java:49)
clojure.lang.RT.seq (RT.java:521)
clojure.core/seq--4357 (core.clj:137)
clojure.core/dorun (core.clj:3024)
clojure.core/doall (core.clj:3039)
clojure.core/doall (core.clj:3039)
```

Probably [here](https://github.com/flosell/lambdacd-git/blob/master/src/lambdacd_git/core.clj#L205) is failing if I read correctly.

It would be awesome to handle this case graciously.

Btw super good job with `lambdacd` in general !

**Comment by flosell** [comment]
Not sure I understood you correctly: This happens if there are no known-hosts files? 

Also, if you already have a good idea on how to fix the issue, you are always welcome to try your hands on a pull request

**Comment by flosell** [comment]
Ah, sorry, I should have read the title of the issue... Never mind my first question!

**Comment by flosell** [comment]
Fixed and released as [`0.2.1`](https://github.com/flosell/lambdacd-git/releases/tag/0.2.1)

**Comment by arichiardi** [comment]
:+1:
